test/support/assertions.py

```diff
@@ -9,13 +9,13 @@
 def assert_valid_schema(data, schema_file):
     """Checks whether the given data matches the schema"""
 
-    schema = _load_json_schema(schema_file)
+    schema = load_json_schema(schema_file)
     schema_dir = join(dirname(__file__), "schemas")
     resolver = RefResolver(base_uri="file://" + schema_dir + "/", referrer=schema)
     return validate(data, schema, resolver=resolver)
 
 
-def _load_json_schema(filename):
+def load_json_schema(filename):
```

Comment:
Maybe other functions should be moved here? (`assert_valid_rpc_schema`)?

Reply:
Not sure about this. Methods added by me are very much RPC specific.

Reply:
My point is that `load_json_schema` was private. But you're right they are connected only with the RPC